### PR TITLE
Add Cinder alerts

### PR DESCRIPTION
**Pull request by kevin-fischer**
Will be sent to the VMware warning channel

- [ ] Implement
- [ ] Test and validate

### CODE REVIEW

**Review by himanip94 — CHANGES_REQUESTED** [review]
Need to add another condition to exempt Reserved DS for CinderTagMissingForDatastoreSSD and CinderTagMissingForDatastoreHDD

**Review by himanip94 — APPROVED** [review]
lgtm

File: prometheus-rules/prometheus-vmware-rules/alerts/datastore.alerts [hunk 1]
```diff
@@ -341,3 +339,54 @@ groups:
...
+      and on (datastore) vrops_datastore_cinder_state{cinder_state!="reserved"}
+    for: 5m
+    labels:
+      severity: info
+      tier: vmware
+      service: storage
+      support_group: compute
+      meta: "There is no Cinder tag present for the `{{ $labels.datastore }}` datastore in the *{{ $labels.storagepod }}* cluster\nLink to the vCenter: --> <{{ $labels.vcenter }}>"
+      playbook: docs/support/playbook/cinder/#
+      no_alert_on_absence: "true"
```
Comment (richardtief) [inline prometheus-rules/prometheus-vmware-rules/alerts/datastore.alerts:355]: @kevin-fischer @himanip94 
```suggestion
```
If you want to see absent alerts for the metrics you are using, this line can be omitted. See [the operator playbook for absent alerts](https://github.com/sapcc/absent-metrics-operator/blob/master/docs/playbook.md). 

This only applies to short-lived metrics such as the `*_alert_info` metric, which only exists when an alert event is captured by the vrops-exporter.

**Review by richardtief — COMMENTED** [review]
I love seeing how you adapt to our workflow. Keep it up!

File: prometheus-rules/prometheus-vmware-rules/alerts/datastore.alerts (hunk 1 above)
Comment (kevin-fischer) [reply prometheus-rules/prometheus-vmware-rules/alerts/datastore.alerts:355]: Good to know.
I've made the changes and will be pushing them to another open PR with further additions.
Thx a lot Richard!

**Pull request merged**